GOVERNANCE.md

```diff
@@ -1,12 +1,34 @@
 # Governance
 
-## License
+## GUAC community
 
-The project will use [Apache 2.0 license](LICENSE) for code.
+The GUAC community (including GUAC OpenSSF meetings, website  and [github organization](https://github.com/guacsec/) administration) is home to several projects focused around solving problems related to the consumption and utilization of supply chain metadata at scale. The community contains "Core projects" which emcapsulate large bodies of work where a separate governance of code and process makes sense, while retaining requirements around aspects such as licensing and code of conduct.
```

Comment:
Additional responsibilities of what the guac community governance group will do - define which projects be in different core projects. 

Community aspects

Reply:
what are focuses of the group - what are topics we should tackle, where should we bring certain projects and groups together to work on certain problems.